### PR TITLE
Dummy seed

### PR DESCRIPTION
**Pull request by KAYEMBAJONATHAN**
- Add update the seed dummy

### CODE REVIEW

**Review by MathildaShongwe15 — COMMENTED** [review]
I think you forgot to correct the ID Numbers and the Cellphone Number

**Review by bseptember — APPROVED** [review]

**Comment by bseptember** [comment]
@KAYEMBAJONATHAN @MathildaShongwe15 @ReitumetseMoloto We are just holding off on this until this ticket is complete:
The reason being the employee table will be updated and the postal table will be removed, therefore the seed data will no longer be valid. Yielding errors.


[Azure board](https://dev.azure.com/retro-rabbit/HRIS/_sprints/taskboard/HRIS%20Team/HRIS/Sprint%2026?workitem=10082)

**Review by bseptember — CHANGES_REQUESTED** [review]
@KAYEMBAJONATHAN @MathildaShongwe15 @ReitumetseMoloto We are just holding off on this until this 
[ticket](https://dev.azure.com/retro-rabbit/HRIS/_sprints/taskboard/HRIS%20Team/HRIS/Sprint%2026?workitem=10082) is complete:
The reason being the employee table will be updated and the postal table will be removed, therefore the seed data will no longer be valid. Yielding errors.

File: RR.UnitOfWork/SEED_DUMMY.sql [hunk 1]
```diff
@@ -266,51 +257,49 @@ INSERT INTO "RoleAccessLink" (id, "roleAccessId", "roleId")
...
-VALUES (2, '085456565656', NULL, 'SA', TIMESTAMPTZ '1996-12-09 22:00:00Z', FALSE, 'na', 'amanders@gmail.com', NULL, NULL, '8464', 7, TIMESTAMPTZ '2024-03-07 13:10:48.9309Z', 1, NULL, '0231646', 'A', 1, 4, 'Alicia', 'South African', '', NULL, NULL, NULL, 1, NULL, 'fake.email@gmail.com', '', 1, 2, 1, 5, 30, 'Manders', '8465468', NULL, NULL,True,NULL);
+VALUES (2, '+227716480198', NULL, 'Australia', TIMESTAMPTZ '1996-12-09 22:00:00Z', FALSE, 'na', 'amanders@gmail.com', NULL, NULL, '0231646', 7, TIMESTAMPTZ '2024-03-07 13:10:48.9309Z', 1, NULL, '9902027144185', 'A', 1, 4, 'Alicia', 'South Africa', '', NULL, NULL, NULL, 1, NULL, 'fake.email@gmail.com', '', 1, 2, 1, 5, 30, 'Manders', '8465468', NULL, NULL,True,NULL);
```
Comment (bseptember) [inline RR.UnitOfWork/SEED_DUMMY.sql]: The work email should all be username@retrorabbit.co.za, instead of amanders@gmail.com, check all the emails.

**Review by bseptember — CHANGES_REQUESTED** [review]

**Review by MathildaShongwe15 — APPROVED** [review]

**Review by bseptember — APPROVED** [review]

**Review by madwizzard — APPROVED** [review]

**Pull request merged**